doc/devices/drivers/gpio/gpio_stm32.rst

```diff
@@ -0,0 +1,316 @@
+..
+    Copyright (c) 2017 Bobby Noelte
```

Comment:
We haven't been putting copyright or SPDX tags on documentation...

Reply:
When writing the documentation for pinctrl I had a major problem to figure out whether I can re-use the pinctrl docs from linux. In the end, after your advise in an email, I just reference the documentation and otherwise somehow re-invented the wheel. I would not mind somebody else re-using my documentation. So I pledge to have the copyright notice also in the documentation. Al least not to remove it in my documentation.